addons/account/tests/test_account_payment.py

```diff
@@ -903,3 +903,105 @@ def test_internal_transfer_right_accounts(self):
             {'account_id': bank_2.inbound_payment_method_line_ids.payment_account_id.id},
             {'account_id': transfer_account.id},
         ])
+
+    def test_update_journal_account(self):
```

Comment:
There is already a test `test_payment_move_sync_create_write`. Maybe it would be cleaner to rename this test to `test_payment_move_sync_update_journal_custom_accounts`